lib/view_component/polymorphic_slots.rb

```diff
@@ -25,23 +25,26 @@ def renders_many(slot_name, callable = nil)
       end
 
       def register_polymorphic_slot(slot_name, types, collection:)
+        unless types.empty?
```

Comment:
This change feels unrelated, is it aiming at solving a warning?

Reply:
Yes, the warning message is reported in the commit, but basically the getter method was redefined for each type.
The `unless types.empty?` guard is there just to keep the exact same behavior, i.e. if `types` was empty the getter is not defined.